test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/AudioTranscriptions.cs

```diff
@@ -48,9 +48,9 @@ public virtual async Task<ClientResult<CreateTranscriptionResponse>> CreateAsync
         {
             Argument.AssertNotNull(audio, nameof(audio));
 
-            RequestOptions context = FromCancellationToken(cancellationToken);
+            RequestOptions options = FromCancellationToken(cancellationToken);
```

Comment:
Not sure if you plan to handle this in this or a later PR, but:
1. Convenience methods in unbranded clients should not take CancellationTokens
2. Once 1. is addressed, you can remove the `FromCancellationToken` helper method from unbranded clients

Reply:
Yes remove cancellation token is tracked here https://github.com/Azure/autorest.csharp/issues/4243

Reply:
Sounds good.